landoui/templates/stack/stack.html

```diff
@@ -28,6 +28,9 @@ <h1>Stack containing revision {{revisions[revision_phid]['id']}}</h1>
           <th>Status</th>
           <th>Revision</th>
           <th>Reviewers</th>
+          {% if target_repo.approval_required %}
+          <th>Approval</th>
+          {% endif %}
```

Comment:
the `target_repo` is for the landable series which may just be a small piece of the whole stack. Throughout the stack the various revisions might have distinct repositories. How do things look under this scenario? It'd be great to see some screenshots with various types of stacks.